cloudwatch_monitoring/lookups.py

```diff
@@ -326,7 +326,7 @@
             'identifier': 'observations-qa',
         },
         'PROD-EXTERNAL': {
-            'identifier': 'observations-prod-external-2',
+            'identifier': 'observations-db-legacy-production-external',
```

Comment:
The value required here is just the db instance id.  You can see it in the configuration tab in the aws management console -> RDS -> Databases -> observations-db-legacy-production-external -> configuration tab.